knip.js

```diff
@@ -0,0 +1,40 @@
+export default {
+  // the exclamation mark suffix (!) indicates production file patterns
+  entry: ['src/App/App.tsx!'],
+  project: ['src/**/*.{js,ts,tsx}!'],
+  ignore: ['src/test/**'], // folder for unit testing mocks and stubs
+  ignoreBinaries: ['knip'],
+  ignoreDependencies: [
```

Comment:
I assume we need to stay on top of this list as the project's dependencies change